app/models/game.rb

```diff
@@ -1,11 +1,9 @@
 class Game < ApplicationRecord
-  belongs_to :white_player, :class_name => 'User'
-  belongs_to :black_player, :class_name => 'User'
-  belongs_to :user
-  has_many :pieces
+  belongs_to :white_player_id, class_name: 'User', optional: true
```

Comment:
Maybe this should not be optional since white player will always be there - from the point the game is created...

Reply:
I tried belongs_to :white_player_id, class_name: 'User   and deleted the optional: true but this causes problems where the chess name and board will not show up. I can do a demo for you to see tonight. Maybe it just needs a skillful eye to see the issue.